frontend/src/container/InfraMonitoringK8s/EntityStatusEmptyStateWrapper.tsx

```diff
@@ -0,0 +1,173 @@
+import { Typography } from 'antd';
+import { K8sEntityStatusResponse } from 'api/infraMonitoring/getK8sEntityStatus';
+
+import { K8sCategory } from './constants';
+
+interface EntityStatusEmptyStateProps {
+	category: K8sCategory;
+	data: K8sEntityStatusResponse | null | undefined;
+	children: JSX.Element;
+}
+
+// eslint-disable-next-line sonarjs/cognitive-complexity
+function EntityStatusEmptyStateWrapper({
+	category,
+	data,
+	children,
+}: EntityStatusEmptyStateProps): JSX.Element {
+	if (!data) {
+		return children;
+	}
+
+	const {
+		didSendPodMetrics,
+		didSendNodeMetrics,
+		didSendClusterMetrics,
+		isSendingOptionalPodMetrics,
+		isSendingRequiredMetadata,
+	} = data;
+
+	const metaData = isSendingRequiredMetadata.length
+		? isSendingRequiredMetadata[0]
+		: undefined;
+
+	const noK8sMetrics = (
+		<Typography.Text>
+			No k8s metrics were detected. To monitor your k8s infra, you will need to
+			send{' '}
+			<Typography.Link href="https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver">
+				kubelet
+			</Typography.Link>{' '}
+			and{' '}
+			<Typography.Link href="https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver">
+				k8scluster
+			</Typography.Link>{' '}
+			metrics. Check out our k8s infra installation guide{' '}
+			<Typography.Link href="https://signoz.io/docs/tutorial/kubernetes-infra-metrics/">
+				here
+			</Typography.Link>{' '}
+			to get started. Reach out to support if you need further assistance.
+		</Typography.Text>
+	);
+
+	const partialMetrics = (
+		<Typography.Text>
+			Partial metrics detected. The following metrics from the kubelet metrics are
+			not received. They help monitor the resource utilisation to their
+			requests/limits. Learn more{' '}
+			<Typography.Link href="placeholder">here</Typography.Link> about how to send
```

Comment:
Waiting to get the links for a couple of places here. Will update these before merging.